src/main/java/at/gammastrahlung/monopoly_server/game/Player.java

```diff
@@ -9,6 +9,7 @@
 import java.util.Objects;
 import java.util.UUID;
 
+
```

Comment:
Very minor: Please avoid unnecessary changes to files that have nothing to with the feature